src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
Yes, I wanted to reorganize folders structure now - in fact, all components in `components` folder are used once and I think it's better moved them to `containers` which are using them. Then we end with just one `components` folder anyway.

> `data, images` will move to `lib`

you mean to `dist` folder? Or ceate new `lib`?